src/resources/js/tickets.js

```diff
@@ -550,6 +550,8 @@ var ticketHeaderImage = window.ticketHeaderImage || {};
 							return;
 						}
 
+						console.log(response.data);
+
```

Comment:
is this intentionally left here? seems like something that should be removed; remove if so.